test/fixtures/sidecars/volume-mounts.yaml

```diff
@@ -0,0 +1,35 @@
+---
+name: volume-mounts
+env:
+  - name: DATACENTER
+    value: foo
+  - name: FROM_INJECTOR
+    value: bar
+containers:
+- name: sidecar-nginx
+  image: nginx:1.12.2
+  imagePullPolicy: IfNotPresent
+  env:
+    - name: DATACENTER
+      value: bf2
+  ports:
+    - containerPort: 80
+  volumeMounts:
+    - name: nginx-conf
+      mountPath: /etc/nginx
+    - name: test-vol
+      mountPath: /tmp/test
+- name: another-sidecar
+  image: foo:69
+  ports:
+    - containerPort: 420
+  volumeMounts:
+    - name: test-vol
+      mountPath: /tmp/another-dir
+volumes:
```

Comment:
this test fixture does not actually include any `volume-mounts:` keys. You are hardcoding your volumeMounts into your containers instead.

Reply:
Yes, I think I misunderstood the naming here for testing :D
What I thought was: the objects in `test/fixtures/k8s/` are requesting the configmaps in the same directory and the stuff in `test/fixtures/sidecars/` is what should be the result after injection... That got me quite confused and this is also why I didn't use the existing "count number of elements" tests for the volumeMounts.

Could you maybe explain how the test files fit together? :)

Reply:
* `test/fixtures` - any test fixture
* `test/fixtures/k8s` - manifests that are valid k8s objects
* `test/fixtures/sidecars` - k8s-sidecar-injector configuration files that define a sidecar configuration (hardcoded YAML files, these are functionally identical to what is in the ConfigMaps the service reads at runtime)